brainscore/metrics/accuracy_delta.py

```diff
@@ -0,0 +1,108 @@
+import itertools
+
+import numpy as np
+
+from brainscore.metrics import Metric, Score
+import random
+from brainio.assemblies import BehavioralAssembly
+
+
+# controls how many half-splits are averaged together to get human delta.
+HUMAN_SPLITS = 100
+
+
+class AccuracyDelta(Metric):
+
+    def __init__(self, image_types: list):
+        self.image_types = image_types
+
+    def __call__(self, source: BehavioralAssembly, target: BehavioralAssembly):
+
+        model_delta = get_model_delta(source, self.image_types)
+        scores = []
+
+        # calculate score over average of 100 sub splits of human delta
+        for i in range(HUMAN_SPLITS):
+            human_delta = get_human_delta(target, self.image_types, random_state=i)
+            score = max((1 - ((np.abs(human_delta - model_delta)) / human_delta)), 0)
+            scores.append(score)
+        score = np.mean(scores)
+        error = np.std(scores)
+
+        return Score([score, error], coords={'aggregation': ['center', 'error']}, dims=('aggregation',))
+
+
+def extract_subjects(assembly):
+    return list(sorted(set(assembly['subject'].values)))
+
+
+def get_human_delta(target, image_types, random_state=0, isCeiling=False):
+    # calculate human accuracies for [whole, condition]
+    condition_scores_human = {}
+
+    if isCeiling:
+        half1 = target
```

Comment:
why do we use all subjects when computing the ceiling? I generally don't like that we're using different code for ceiling compute vs metric compute here, any differences should be outside this method intuitively